css-scroll-snap-2/Overview.bs

```diff
@@ -470,3 +480,114 @@ Physical Longhands for 'scroll-start-target' {#scroll-start-target-longhands-phy
 	</pre>
 
 	...
+
+Appendix B: Snap Events Timing {#snap-events-timing}
+==================================
+
+This appendix describes the algorithm UAs should follow to dispatch {{snapchanged}}
```

Comment:
This is a patch to cssom-view scrolling, correct? https://www.w3.org/TR/cssom-view-1/#scrolling-events Can you make it explicit that we are patching those algorithms, and specifically that they run before the related steps mentioned there.

I think ... though @tabatkins can confirm, that it may be preferred to list all modifications to cssom-view-1 here so that they can be reviewed as part of css-scroll-snap-2.

Reply:
I've included text saying that that section is "a patch to [scrolling events](https://www.w3.org/TR/cssom-view-1/#scrolling-events)"  and pointed out within the algorithm that the steps firing snap events should be run before the steps firing the scroll/scrollend events.
All the text added to cssom-view can be found in this appendix as well but I'm not sure if there is a particular format for representing a "patch"?

Reply:
Linking to the relevant spec and explaining at which step the new algorithms run is sufficient. It doesn't need to be a literal patch. That said, if you are adding directly to cssom-view in this PR then we don't need the appendix section here.

> All the text added to cssom-view can be found in this appendix as well

I think we should only add this text in one place. Either here or in cssom-view. We shouldn't repeat the algorithm in both places as there should be one place in which it is defined.

I think the simplest would be to have a single step in cssom-view run an algorithm in this spec. This is similar for example to how [update animations and send events is defined in web-animations-1](https://www.w3.org/TR/web-animations-1/#update-animations-and-send-events) but [called from the html spec](https://html.spec.whatwg.org/#:~:text=For%20each%20doc%20of%20docs%2C%20update%20animations%20and%20send%20events%20for%20doc%2C%20passing%20in%20relative%20high%20resolution%20time%20given%20frameTimestamp%20and%20doc%27s%20relevant%20global%20object%20as%20the%20timestamp%20%5BWEBANIMATIONS%5D)

Reply:
That makes sense, I've taken out the appendix section and put the algorithms in the snapchanged and snapchanging sections and linked to them from the relevant cssom-view steps.